Botkit/Microsoft.BotKit/BotkitDialogWrapper.cs

```diff
@@ -1,51 +1,56 @@
-// Copyright(c) Microsoft Corporation.All rights reserved.
-// Licensed under the MIT License.
-
-using Microsoft.BotKit.Conversation;
-using Microsoft.Bot.Builder.Dialogs;
-using System;
+﻿// <copyright file="BotkitDialogWrapper.cs" company="PlaceholderCompany">
+// Copyright (c) PlaceholderCompany. All rights reserved.
+// </copyright>
```

Comment:
Please, leave the copyright header as it was.

Reply:
Done